### ISSUE TITLE
Error generating detection model

### ISSUE THREAD

**Issue by nnatalieyn** [Jordan-Pierce/CoralNet-Toolbox#211]
### Question

I got an error message trying to generate an Ultralytics detection model. I haven't done anything differently from before, besides just reusing an exported training dataset I had previously made, instead of making a new training dataset from scratch. However I experimented more and created a new dataset to test this out, as well as inputted every dataset I previously generated, and still got the error.
The error log seem to suggest that the I am missing the pathway to valid/image folder, but all my datasets include this folder.
Any guidance? Thanks again!

### Context

Error log:
``
Error during training: Dataset 'C://Users/Documents/exported datasets/detect_data_tiled2/data.yaml' error  Dataset 'C://Users/Documents/exported datasets/detect_data_tiled2/data.yaml' images not found, missing path 'C:\Users\Documents\CoralNet-Toolbox\datasets\detect_data_tiled2\detect_data_tiled2\valid\images'
Note dataset download directory is 'C:\Users\Documents\CoralNet-Toolbox\datasets'. You can update this in 'C:\Users\Flash\AppData\Roaming\Ultralytics\settings.json' (see console log)
``

**Comment by Jordan-Pierce** [comment]
Hey @nnatalieyn which version are you currently using, and did you recently switch to a new version that might be related to the new version?

If not, tell me more:
- you're trying to train a detection / segmentation model, or a classification model?
- the error is complaining about your yaml file: have you moved any data around or accidently overwritten a yaml file?

`C://Users/Documents/exported datasets/detect_data_tiled2/data.yaml' images not found, missing path`

If you open up the yaml file, you'll see the paths for the train / valid / test sets: do this currently point to valid paths? (i.e., are the images and labels where they ought to be)

**Comment by Jordan-Pierce** [comment]
Closing this now due to inactivity. Let me know if you're still experiencing issues.
